docs/api-docs/getting-started/api-status-codes.html

```diff
@@ -0,0 +1,262 @@
+</style>
+        <style>
+            body {
+                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', 'Ubuntu', 'Droid Sans', sans-serif;
+                font-size: 14px;
+                line-height: 1.6;
+            }
+        </style>
+        
+        <script src="https://cdn.jsdelivr.net/npm/katex-copytex@latest/dist/katex-copytex.min.js"></script>
+    </head>
+    <body>
+        <h1 id="api-status-codes">API Status Codes</h1>
+<div class="otp" id="no-index">
+	<h3> On This Page </h3>
+	<ul>
+        <li><a href="#api-status-codes_2-success">2xx Success</a></li>
+        <li><a href="#api-status-codes_3-redirection">3xx Redirection</a></li>
+        <li><a href="#api-status-codes_4-client-error">4xx Client Error</a></li>
+        <li><a href="#api-status-codes_5-server-error">5xx Server Error</a></li>
+        <li><a href="#api-status-codes_troubleshooting">Troubleshooting</a></li>
+	</ul>
+</div>
+<p>The BigCommerce API responds to requests with different HTTP status codes depending on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.</p>
+<h2 id="2xx-success">2xx Success</h2>
+<p>2xx codes are returned for requests that were understood and processed successfully.</p>
+<table>
+<thead>
+<tr>
+<th>Code</th>
+<th>Text</th>
+<th>Purpose</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><strong>200</strong></td>
+<td><strong>OK</strong></td>
+<td>For successful <code>GET</code> and <code>PUT</code> requests.</td>
+</tr>
+<tr>
+<td><strong>201</strong></td>
+<td><strong>Created</strong></td>
+<td>For a successful <code>POST</code> request.</td>
+</tr>
+<tr>
+<td><strong>202</strong></td>
+<td><strong>Accepted</strong></td>
+<td>For a request that resulted in a scheduled task being created to perform the actual request.</td>
+</tr>
+<tr>
+<td><strong>204</strong></td>
+<td><strong>No Content</strong></td>
+<td>For a successful request that produced no response (such as <code>DELETE</code> requests).</td>
+</tr>
+</tbody>
+</table>
+<h2 id="3xx-redirection">3xx Redirection</h2>
+<p>3xx codes are returned for requests that require further action.</p>
+<table>
+<thead>
+<tr>
+<th>Code</th>
+<th>Text</th>
+<th>Purpose</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><strong>301</strong></td>
+<td><strong>Moved Permanently</strong></td>
+<td>When the API routes have changed (unlikely), or if the incoming request is not secure (<code>http</code>), the request will be redirected to the secure (<code>https</code>) version.</td>
+</tr>
+<tr>
+<td><strong>304</strong></td>
+<td><strong>Not Modified</strong></td>
+<td>This response will be sent if the request included an <code>If-Modified-Since</code> header, but the resource has <em>not</em> been modified since the specified date. Please refer to individual resources' documentation regarding support for the <code>If-Modified-Since</code> header.</td>
```

Comment:
What is "individual resources'" referring to here? Should it be "individual resource's" instead? Should it be linked?